addons/hr_attendance/static/src/js/greeting_message.js

```diff
@@ -106,7 +106,7 @@ var GreetingMessage = AbstractAction.extend({
         if(this.previous_attendance_change_date){
             var last_check_in_date = this.previous_attendance_change_date.clone();
             if(now - last_check_in_date > 1000*60*60*12){
-                this.$('.o_hr_attendance_warning_message').append(_t("Warning! Last check in was over 12 hours ago.<br/>If this isn't right, please contact Human Resources."));
+                this.$('.o_hr_attendance_warning_message').show().append(_t("<b>Warning! Last check in was over 12 hours ago.</b><br/>If this isn't right, please contact Human Resources."));
```

Comment:
Definitively, thanks

Reply:
Actually... I had a [look online](https://www.linguee.com/english-french/search?source=auto&query=contact+a+human+resource) and seems like the **a** article is used in conjunction with the person position only, eg:
[*"contact a human resources **representative**"* ](https://www.linguee.com/english-french/search?query=contact+a+human+resource+representative&source=auto)

I think that I'll go this, because fits better the available room:

"If this isn't right, please contact Human Resource **staff**"